### PR TITLE
Auto accept android license on build machines.

### PR DESCRIPTION
**Pull request by damencho**
Every time the android version that is used changes we need to update the license hash.

### CODE REVIEW

File: android/build.gradle [hunk 1]
```diff
@@ -141,6 +141,47 @@ ext {
...
+    }
+    // If the license is not accepted, accept it so that automatic downloading
+    // kicks in.
+    if (androidHome) {
+        def dir = file("${androidHome}/licenses")
+        dir.mkdirs()
+        def file = file("${dir.path}/android-sdk-license")
+        if (!file.exists()) {
+            file.withWriter {
+                def hash = 'd56f5187479451eabf01fb78af6dfcb131a6481e'
```
Comment (saghul) [inline android/build.gradle]: can you add a comment on how to get these hashes, should they change?

Comment (damencho) [reply android/build.gradle]: I've added something, not sure is it enough.

**Comment by damencho** [comment]
Ooo I need this one merged to be able to build APKs when doing PR testing.

**Comment by damencho** [comment]
Ignore my last comment, it is not true.

**Comment by damencho** [comment]
Ups, too quick. At the end it needs it :)

**Pull request merged**